crates/sncast/src/helpers/configuration.rs

```diff
@@ -0,0 +1,102 @@
+use super::constants::CONFIG_FILENAME;
+use anyhow::{anyhow, Result};
+use camino::Utf8PathBuf;
+use std::fs;
+
+pub fn find_config_file_relative_to(current_dir: &Utf8PathBuf) -> Result<Utf8PathBuf> {
+    current_dir
+        .ancestors()
+        .find(|path| fs::metadata(path.join(CONFIG_FILENAME)).is_ok())
+        .map(|path| path.join(CONFIG_FILENAME))
+        .ok_or_else(|| {
+            anyhow!(
+                "Failed to find snfoundry.toml - not found in current nor any parent directories"
+            )
+        })
+}
+
+pub fn find_config_file() -> Result<Utf8PathBuf> {
```

Comment:
And here we would could leave name as it is to abstract the config searching process.